src/lib/y2storage/proposal_settings.rb

```diff
@@ -50,6 +50,10 @@ def initialize
       @home_filesystem_type     = ::Storage::FsType_XFS
       @enlarge_swap_for_suspend = false
     end
+
+    def use_encryption
```

Comment:
Yes, then what is it better? Rename use_encryption as encrypt? or rename both, use_lvm and use_encryption to question mark versions?